docs/pages/button.js

```diff
@@ -259,245 +229,481 @@ card(
-  </Box>
-</React.Fragment>`}
-  />,
-);
+  React.useEffect(() => {
+    setOpen(true)
+  });
 
-card(
-  <Example
-    name="Icons"
-    id="iconEnd"
-    defaultCode={`
-    <Button iconEnd="download" text="Download CVS file" />
+  return (
+    <ScrollBoundaryContainer>
+      <Box ref={viewRef} width={300} height={220}>
+        <Box display="flex" justifyContent="center" ref={anchorRef}>
+        <Tabs
+          activeTabIndex={1}
+          onChange={() => {}}
+          tabs={[{ href: '#Anchor', text: 'Created'}]}
+        />
+        </Box>
+      </Box>
+      {open &&
+        <Layer>
+          <Popover
+            anchor={anchorRef.current}
+            color="blue"
+            idealDirection="down"
+            showCaret
+            onDismiss={() => {}}
+            positionRelativeToAnchor={false}
+            size={240}
+          >
+            <Box padding={3}>
+              <Flex alignItems="center" direction="column" gap={4}>
+                <Text color="white" align="center">
+                  New look! Click Created to see Pins you've published. Click Saved to see your saved Pins and boards.
+                </Text>
+                <Flex alignItems="stretch" direction="column" gap={2}>
+                  <Button
+                    color="white"
+                    size="lg"
+                    text="Got it"
+                    fullWidth
+                  />
+                  <Button
+                    color="semiTransparentWhite"
+                    size="lg"
+                    text="Learn more"
+                    fullWidth
+                  />
+                </Flex>
+              </Flex>
+            </Box>
+          </Popover>
+        </Layer>}
+    </ScrollBoundaryContainer>
+  )
+}
 `}
-  />,
-);
-
-card(
-  <Example
-    name="Selected state"
-    id="selected"
-    defaultCode={`
-function Example() {
-  const [selected, setSelected] = React.useState(false);
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="Icons"
+      description={`
+1. Icon end
+Adds an icon after the Button text. Icons should only be used to reinforce a specific function or interaction of the Button. Menus and external links are a common use case. Use \`arrow-up-right\` when linking to an external URL or \`arrow-down\` when displaying a Popover on click. Note that iconEnd on Button is not accessible to screen readers.
+`}
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+<Button
+  accessibilityLabel='Menu'
+  iconEnd="arrow-down"
+  size="lg"
+  text="Menu"
+/>
+`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      columns={2}
+      title="Role"
+      description={`
+1. Button (default)
+    The “button” \`role\` is used for actions. This is the default and should be used for most Buttons.
+2. Link
+    The “link” \`role\` is used for navigating by URL. These Buttons should not use a \`selected\` state.
```

Comment:
VOILA!!! Check this conversation https://pinterest.slack.com/archives/G01GN7CE99S/p1631231159047700

Reply:
I can't see the conversation. :(